git-extra/vimrc

```diff
@@ -37,6 +37,4 @@ if has("autocmd")
       autocmd Filetype diff
       \ highlight WhiteSpaceEOL ctermbg=red |
       \ match WhiteSpaceEOL /\(^+.*\)\@<=\s\+$/
-
-      autocmd Syntax gitcommit setlocal textwidth=74
```

Comment:
Yep, this stems from a time when this was *not* configured automatically.

Reply:
I am pretty sure, Vim has support for the diff filetype for more than 15 years however. But most likely, filetype detection was not enabled back then by default.

Reply:
Ah! Thanks for the clarification. I misunderstood.